index.html

```diff
@@ -50,6 +53,35 @@
     <!-- Latest featured projects end -->
 
     <!-- Client's feedback start  -->
+
+    <div class="container">
+        <div class="d-flex justify-content-center">
+            <div class="row visible">
+                <div class="col-12 col-sm-7 ml-sm-3 col-lg-9 ml-lg-3">
```

Comment:
col-sm-6 tada butu  centre nes ml-sm-3 
col-lg-8 ml-lg-2
viso yra 12 stulpeliu tai jai elementas 8 plocio tai is sonu po 2 kad butu centre

Reply:
Šiek tiek pasislinko, bet vos juntamai